docs/content/en/configuration.md

```diff
@@ -213,6 +213,14 @@ export default {
 }
 ```
 
+### `markdown.tocDepth`
+
+- Type: `Number`
+- Default: `3`
+- Version: **>= v1.11.0**
+
+Define until which heading levels is included in table of contents.
```

Comment:
I'm not sure this sentence is valid in English...

Reply:
I would say: `Maximum heading depth to include in the table of contents` (source: https://github.com/remarkjs/remark-toc#optionsmaxdepth)